.github/workflows/ci-cd.yml

```
@@ -0,0 +1,166 @@
+        
+    - name: Install dependencies
+      run: pnpm install --frozen-lockfile
+      
+    - name: Run unit tests
+      run: pnpm run test
+      
+    - name: Run e2e tests
+      run: pnpm run test:e2e
+      
+    - name: Upload coverage reports
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)